learning_observer/learning_observer/lms_integration.py

```diff
@@ -0,0 +1,287 @@
+import json
+import recordclass
+import string
+import aiohttp
+import aiohttp.web
+import aiohttp_session
+
+import learning_observer
+import learning_observer.runtime
+import learning_observer.google
+import learning_observer.constants as constants
+import learning_observer.settings as settings
+import pmss
+
+pmss.register_field(
+    name="default_server",
+    type=pmss.pmsstypes.TYPES.string,
+    description="The Canvas OAuth default server",
+    required=True
+)
```

Comment:
This is the second place that `default_server` has been registered.

Reply:
Replaced with `lms_api`